lib/tasks/generate_packs.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :react_on_rails do
+  desc <<~DESC
+    If there is a jsx file inside config.components_directory, this command generates corresponding packs and#{' '}
+    registers the component for usage with react_component_with_bundle helper.
```

Comment:
Actually, is this helper added? Because I can't see it anywhere in changes. If it isn't added because it was decided to take another approach, please change this and also update the PR description.